.circleci/config.yml

```diff
@@ -57,6 +57,15 @@ jobs:
       - run:
           name: Publish @next
           command: npm publish --tag next --access public
+      - run:
+          name: Login to Docker Hub
+          command: echo "$DOCKER_PASS" | docker login --username $DOCKER_USER --password-stdin
+      - run: 
```

Comment:
Shouldn't we be building the version with commit hash here as well, the same way we publish to npm a specific beta and then the same version as `@next`?

Reply:
Agreed, updated to pull the version from the package after it is updated by the script which is then passed as a build arg.